Task3/app/src/main/aidl/com/example/task3/IMyAidlInterface.aidl

```diff
@@ -0,0 +1,13 @@
+// IMyAidlInterface.aidl
```

Comment:
this file can be removed.

Reply:
Already removed, only used during aidl learning.